docs/setup/config.md

```diff
@@ -211,6 +211,8 @@ The parameters define the behavior of tasks that create result files.
 	
 	yarn.application-master.env.LD_LIBRARY_PATH: "/usr/lib/native"
 
+- `yarn.containers.vcores` The number of virtual cores (vcores) per YARN container. By default the number of `vcores` is set equal to the maximum between the number of slots per TaskManager, and the number of cores available to the Java runtime. 
```

Comment:
what's the rationale for using the max(slots, #cpus) ?
I think in most cases users use fewer slots than CPU cores available on the physical machine.